README.md

```diff
@@ -1,13 +1,24 @@
 # Express API Project
 
-Replace this readme with your own information about your project. 
+This is a backend project with a purpose to build an API with hardcoded data in a json-file. And create endpoints and using a RESTful approach. The Data used is data over netflix titles both movies and tv shows.
```

Comment:
This is a very descriptive, to-the-point presentation of what your app does. Good one!